ldk/goja/src/clipboard.ts

```diff
@@ -1,28 +1,47 @@
+/**
+ * The ClipboardService provides access to the OS's clipboard.
+ */
 export interface Clipboard {
+
+    /**
+   * @returns A Promise resolving with the current contents of the clipboard.
+   */
   read(): Promise<string>;
-  write(val: string): Promise<void>;
-  listen(cb: (val: string) => void): void;
+
+    /**
+   * Writes the provided text into the clipboard.
+   *
+   * @param text
+   */
+  write(text: string): Promise<void>;
+  
+  /**
+   * Starts listening to changes to the clipboard.
+   *
+   * @param callback - A function that's called whenever the clipboard's contents change.
+   */
+  listen(callback: (clipboardText: string) => void): void;
 }
 
-function listen(cb: (val: string) => void): void {
-  return oliveHelps.clipboard.listen(cb);
+function listen(callback: (clipboardText: string) => void): void {
+  return oliveHelps.clipboard.listen(callback);
 }
 
 function read(): Promise<string> {
   return new Promise<string>((resolve, reject) => {
     try {
-      oliveHelps.clipboard.read((val: string) => resolve(val));
+      oliveHelps.clipboard.read((clipboardText: string) => resolve(clipboardText));
     } catch (e) {
       reject(e);
       // TODO: add console log
```

Comment:
did we want to develop some very basic standard for a console log for these here?

Reply:
I agree, pushed changes.